tanglescope/runner/configuration.yaml

```diff
@@ -3,7 +3,7 @@ dbloader:
   #URL of ZMQ publisher to connect to
   publisher: "tcp://zmq.testnet.iota.org:5556"
   #TX's older than that age will be removed each cleanup_interval [seconds]
-  oldest_tx_age: 600
+  oldest_tx_age: 1800
```

Comment:
is this change on purpose?

Reply:
yes, i want the db to be consistent enough